dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -95,6 +96,7 @@ public class Contentlet implements Serializable, Permissionable, Categorizable,
   public static final String HAS_TITLE_IMAGE_KEY = "hasTitleImage";
   public static final String INODE_KEY = "inode";
   public static final String LANGUAGEID_KEY = "languageId";
+  public static final String VARIANT_ID = "VariantId";
```

Comment:
Change value to "variantId"

Reply:
done https://github.com/dotCMS/core/pull/22933/commits/96edb299e4be6a1b653d61e3b97f90c443ce9061#diff-1367236d40a612b923ececfa521053143f026f60f035c9f9963aae4efdfaf034R99